src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,204 @@
+      mapView.setMap(map);
+
+      // create an Image Service Raster as a raster layer and add to map
+      final ImageServiceRaster imageServiceRaster = new ImageServiceRaster("http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer");
+      final RasterLayer imageRasterLayer = new RasterLayer(imageServiceRaster);
+      map.getOperationalLayers().add(imageRasterLayer);
+
+      // add event listener to loading of Image Service Raster and wait until loaded
+      imageRasterLayer.addDoneLoadingListener(() -> {
+        if (imageRasterLayer.getLoadStatus() == LoadStatus.LOADED) {
+          // zoom to extent of the raster
+          mapView.setViewpointGeometryAsync(imageServiceRaster.getServiceInfo().getFullExtent());
+
+          // get the predefined rendering rules
+          List<RenderingRuleInfo> renderingRuleInfos = imageServiceRaster.getServiceInfo().getRenderingRuleInfos();
+
+          // populate the drop down menu with the rendering rule names
+          for (RenderingRuleInfo renderingRuleInfo : renderingRuleInfos) {
+            // add the name of the rendering rule to the drop-down menu
+            renderingRuleInfoComboBox.getItems().add(renderingRuleInfo);
+          }
+
+          // listen to selection in the drop-down menu
+          renderingRuleInfoComboBox.getSelectionModel().selectedItemProperty().addListener(o -> {
+
+            // get the requested rendering rule info from the list
+            RenderingRuleInfo selectedRenderingRuleInfo = renderingRuleInfoComboBox.getSelectionModel().getSelectedItem();
+
+            // change the label text to the rendering rule info description
+            String renderingRuleInfoDescription = selectedRenderingRuleInfo.getDescription();
+            renderingRuleInfoLabel.setText("Rule Description: \n" + renderingRuleInfoDescription);
+
+            // clear all rasters
+            map.getOperationalLayers().clear();
+
+            // create a rendering rule object using the rendering rule info
+            RenderingRule renderingRule = new RenderingRule(selectedRenderingRuleInfo);
+
+            // create a new image service raster
+            ImageServiceRaster appliedImageServiceRaster = new ImageServiceRaster("http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer");
+
+            // show progress indicator while rule is loading
+            appliedImageServiceRaster.addLoadStatusChangedListener((e)->{
+              if (e.getNewLoadStatus() == LoadStatus.LOADING){
+                progressIndicator.setVisible(true);
+              } else {
+                progressIndicator.setVisible(false);
+              }
+            });
+
+            // apply the rendering rule
+            appliedImageServiceRaster.setRenderingRule(renderingRule);
+            RasterLayer rasterLayer = new RasterLayer(appliedImageServiceRaster);
+            map.getOperationalLayers().add(rasterLayer);
+
+          });
+        }
```

Comment:
can add `renderingRuleInfoComboBox.getSelectionModel().selectFirst();` in here to show the user the displayed raster on loading is with the rule "none".